### PR TITLE
Moves the settings section into a scene to correspond with the new edit profile, and selling overview

### PR DESCRIPTION
**Pull request by orta**
Mainly just fixing the storybooks and moving some files around.

#trivial

### CODE REVIEW

File: src/lib/Containers/__stories__/MyProfile.story.tsx [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-// import React from "react"
-// import * as Relay from "react-relay/classic"
-
-// import Routes from "../../relay/routes"
-// import MyProfile from "../MyProfile"
-
-// storiesOf("My Account").add("Root", () => {
-//   const profileRoute = new Routes.MyProfile()
-//   return <Relay.RootContainer Component={MyProfile} route={profileRoute} />
-// })
```
Comment (orta) [inline src/lib/Containers/__stories__/MyProfile.story.tsx]: There's a _lot_ of these i the codebase I discovered, I swear that the GMV iOS team avoids storybooks :P

File: src/lib/Scenes/Settings/Selling.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    }
+  `
+)
+
+interface RelayProps {
+  me: {
+    name: string | null
+    initials: string | null
+  }
+}
```
Comment (orta) [inline src/lib/Scenes/Settings/Selling.tsx]: Hrm, I'll delete this, it's just a dupe of the My Profile page

**Comment by orta** [comment]
assigning to you instead ash

**Comment by mennenia** [comment]
Looks good! Want me to merge, or rather wait for ash? @orta

**Comment by orta** [comment]
Nah - I'll do it now

**Pull request merged**